nfConfigApi/model_direction.go

```diff
@@ -25,10 +25,10 @@ type Direction string
 
 // List of Direction
 const (
-	DIRECTION_DOWNLINK      Direction = "DOWNLINK"
-	DIRECTION_UPLINK        Direction = "UPLINK"
-	DIRECTION_BIDIRECTIONAL Direction = "BIDIRECTIONAL"
-	DIRECTION_UNSPECIFIED   Direction = "UNSPECIFIED"
+	DOWNLINK      Direction = "DOWNLINK"
+	UPLINK        Direction = "UPLINK"
+	BIDIRECTIONAL Direction = "BIDIRECTIONAL"
+	UNSPECIFIED   Direction = "UNSPECIFIED"
```

Comment:
Why is this change needed? I think it is better to have the `DIRECTION_` prefix such as there is no confusion with other const that may have the same value.
This comment is applicable in all files where there are `const` defined